modules/quickstart-public-cloud/pages/setup.adoc

```diff
@@ -6,18 +6,57 @@ Public clouds provide {productname} under a Bring Your Own Subscription (BYOS) m
 This means that you must register instances with the {scc}.
 For more information about registering {productname} with {scc}, see xref:installation:general-requirements.adoc[].
 
-Depending on the public cloud framework you are using, you can locate the {productname} images by searching for the  keywords ``suse``, ``manager``, ``proxy``, or ``BYOS``.
-Select a public cloud instance type that meets the hardware requirements in xref:installation:hardware-requirements.adoc[].
+Depending on the public cloud framework you are using, you can locate the {productname} images by searching for the  keywords ``suse``, ``manager``, or ``BYOS``.
+
+.SUSE Manager Image Names
+[cols="1,1", options="header"]
+|===
+| Provider              | Image Name
+| Amazon EC2            |
+| Google Compute Engine |
+| Microsoft Azure       |
+|===
+
+Select a public cloud instance type with at least:
+
+* 32{nbsp}GB RAM
+* 8 cores
+
+If you are not using the instance for production, then you will require at least:
+
```

Comment:
There is not any difference 😉 Maybe, for simple tests 16GB and 4 cores are enough.  To be approved by someone else.